### PR TITLE
Update Db2LinkedServiceTypeProperties

### PR DESCRIPTION
**Pull request by Jingshu923**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json [hunk 1]
```diff
@@ -1210,6 +1210,14 @@
...
           "$ref": "../datafactory.json#/definitions/SecretBase"
         },
+        "packageCollection": {
+          "type": "object",
+          "description": "Under where packages are created when querying database. Type: string (or Expression with resultType string)."
+        },
+        "certificateCommonName": {
+          "type": "object",
+          "description": "Certificate Common Name when TLS is enabled. Type: string (or Expression with resultType string)."
+        },
```
Comment (anthony-c-martin) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json]: Adding properties to a stable API version is typically regarded as a breaking change, since older versions of the client SDKs will be unable to read them. What is the reason for this change?

**Review by anthony-c-martin — CHANGES_REQUESTED** [review]

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json (hunk 1 above)
Comment (Jingshu923) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json]: Although they are new properties but the older versions of the client SDKs still be able to read them. It is just a simple update with some new features.

Comment (weidongxu-microsoft) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json]: Make sure GET-PUT pipeline is good for old client. I.e., if new param is defined in resource, but old client GET the resource (not able to parse the new param), then PUT it back, it will not erase that new param previous defined.

https://microsoft.sharepoint.com/:w:/r/teams/azureresourcemanagerteam/_layouts/15/guestaccess.aspx?rtime=oPAFQY8Z10g&share=EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw

**Review by weidongxu-microsoft — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
@anthony-c-martin

Hello, if you are OK with Jianshu's explanation would you change the reivew status to approve, so I could merge the PR? Thanks.

**Review by anthony-c-martin — APPROVED** [review]
